org.opent2t.onboarding.wifi.flux/js/tests/test.js

```diff
@@ -0,0 +1,29 @@
+var test = require('ava');
+var Onboarding = require("../thingOnboarding");
+
+/** Discovers a Flux light Bulb
+ * This test requires a physical flux bulb for detection.
```

Comment:
>his test requires a physical flux bulb for detection. [](start = 4, length = 53)

We should consider mocking for this to be a true unit test. Can we not add a fake registration similar to Flux  that the discover() module can discover?